docs/reference/usage-metrics.md

```diff
@@ -0,0 +1,48 @@
+---
+id: usage-metrics
+title: "Usage metrics"
+description: "Learn about the three main usage metrics that impact Camunda Platform 8 pricing."
+---
+
+There are three main usage metrics that have an impact on Camunda Platform 8 pricing. It is important to understand these metrics and how to retrieve them.
+
+## Definition of metrics
+
+### Root process instance
+
+The number of **root process instance** executions started. This is also known as process instances (PI). A **root process instance** has no parent process instance, i.e. it is a top-level execution.
+
+### Decision instance
+
+The number of evaluated **decision instances** (DI). A **decision instance** is a DMN decision table or a DMN literal expression. In a DRD each evaluated decision table or expression is counted separately.
```

Comment:
Is it worth linking to process instance details, DMN, or task user documentation for clarity, or is it assumed if a user is visiting this page, they should be aware of these concepts?

Reply:
This is a good question - I'm not sure. Let's add a few links and iterate in the future if we get any feedback.

Reply:
👍